Sources/Clock.swift

```diff
@@ -32,7 +42,17 @@ public struct Clock {
 
     /// The most accurate date that we have so far (nil if no synchronization was done yet)
     public static var now: Date? {
-        return self.timestamp.map { Date(timeIntervalSince1970: $0) }
+        return self.annotatedlNow?.0
```

Comment:
I guess you don't need this change now? alternatively use `.data` as opposed to `.0`

Reply:
good catch, thx